.github/CODEOWNERS

```diff
@@ -15,10 +15,10 @@
 /tests/modules/javascript                   @aristath @gziolo
 /tests/testdata/modules/javascript          @aristath @gziolo
 
-# Focus: Object Caching
-/modules/object-caching                     @tillkruss @dustinrue
-/tests/modules/object-caching               @tillkruss @dustinrue
-/tests/testdata/modules/object-caching      @tillkruss @dustinrue
+# Focus: Object Cache
+/modules/object-cache                       @tillkruss @spacedmonkey
```

Comment:
+1 - let's make sure to also update this elsewhere though, e.g. the GitHub label and our focus area organization spreadsheet.

Reply:
I've already updated the labels and project in the repo. @bethanylang Where else would this need updating?

Reply:
@tillkruss Just the focus spreadsheet; updated here: https://docs.google.com/spreadsheets/d/16N5oZ9wE6AkiqMz7b_707eh24vvpjMwsEG67XFAbxy8/edit#gid=0.